packages/sdk/src/utils/constants.ts

```diff
@@ -0,0 +1,80 @@
+import { ethers } from 'ethers'
+import {
+    L1ChainID,
+    L2ChainID,
+    L1Contracts,
+    L2Contracts,
+    ContractsLike
+} from '../interfaces'
+
+// Docker
+// L1_RPC_URL = http://l1-geth:8545
+// L2_RPC_URL = http://sp-geth:4011
+
+const l1portalAddresses = {
+    sepolia: '0x457818a300BBa8E998889FAb2e8453BDebA238E0',
```

Comment:
these are not stable yet. use a configuration file like json or env file instead of hard-encoding them

Reply:
leaving these like this the way they are, it's just the SDK only in the constants.ts file